src/test/resources/org/rust/ide/inspections/fixtures/cstring_pointer.rs

```diff
@@ -0,0 +1,8 @@
+use std::ffi::{CString};
+
+fn main() {
+    let ptr = <warning descr="Unsafe CString pointer">CString::new("Hello").unwrap().as_ptr()</warning>;
+    unsafe {
+        *ptr;
+    }
```

Comment:
Let's add a safe variant here as well, to make the test serve as a documentation:

```
let s = CString::new("Hello").unwrap();
let ptr1 = s.as_ptr();
let ptr2 = <warning descr="Unsafe CString pointer">CString::new("World").unwrap().as_ptr()</warning>;
unsafe {
     *ptr1;
     *ptr2;
}
```